src/events/guildMemberRemove.js

```diff
@@ -0,0 +1,40 @@
+const { Events, EmbedBuilder } = require("discord.js");
+const moment = require("moment");
+
+/*
+	Event: guildMemberRemove
+	Uses Database?: No
+	Description: Emitted whenever a member leaves a guild, or is kicked.
+*/
+
+module.exports = {
+  name: Events.GuildMemberRemove,
+
+  execute(member) {
+    // Ignore bots.
+    if (member.user.bot) {
+      return;
+    }
+
+    // getting channel info
+    const channel = member.guild.channels.cache.get(
+      process.env.GENERAL_CHANNEL_ID
+    );
+
+    // creating leaving embed
+    const leavingEmbed = new EmbedBuilder()
+      .setColor("00FF00")
+      .setTitle("Member Joined")
```

Comment:
This should be changed to Member Left the Server or something similar.

Reply:
Corrected.